src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -17,6 +19,21 @@ import { errorSchemaIsValid } from '../validation';
 const Element = Scroll.Element;
 const scroller = Scroll.scroller;
 
+
+const isEmpty = (collection) => {
+  if (collection === undefined) {
+    return true;
+  }
+  if (typeof collection !== 'object') {
+    return false;
+  }
+  return _.every(isEmpty, Object.values(collection));
+};
+
+const isInvalid = (item, index, errorSchema) => {
+  return isEmpty(item) || !errorSchemaIsValid(errorSchema[index]);
```

Comment:
What scenario did you hit that requires the `isEmpty` check? It seems like it would only be necessary in cases where an array had a `minItems` requirement, but none of the fields in the array items were required, which I don't believe we do anywhere (because it'd lead to bad data).

Reply:
Yeah, thinking through this, I can't think of a time where this will do what we want it to. If we have an array of empty items, it should be fine unless the objects have required fields. Alternatively, if we had an empty array, that should be fine unless there's a `minItems` (in which case, the `errorSchema` won't be valid).

This here would catch if `minItems > 0` and those items were all blank and those items had no required fields.

Reply:
Thanks @cvalarida and @jbalboni, good point, `isEmpty` removed.